recipes/modelx/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2017-2022 Fumito Hamamura <fumito.ham@gmail.com>
```

Comment:
Both LICENSE.txt and LICENSE.lesser.txt are included in the pypi package. No need to copy that file here.

Reply:
baee2f14f9ba8c04ea2c3bd6022ee97c00b09244 removes the file.